README.md

```diff
@@ -125,7 +140,20 @@ Current state | Next word | Next state
 
 ## A2.4
 
-«replace this with your answer»
+Current state | Next word | Next state
+--------------|-----------|-----------
+    S0        |    the    |     S1  
+    S1        |           |     S1  
+    S1        |    lazy   |     S1  
+    S1        |   smelly  |     S2  
+    S2        |    dog    |     S2  
+    S2        |    cat    |     S3  
+    S3        |    ate    |     S3  
+    S3        |    ran    |     S4  
+    S4        |           |     S4  
+    S4        |   slowly  |     S4  
+    S4        |  noisily  |     S5  
+    S5        |    EOI    |     END  
```

Comment:
# 5

This table seems to be an accurate transcription of your diagram. But the blank entries in the "next word" column should have tipped you off that something was not right...